src/server/enterprise/server/api_server.go

```diff
@@ -0,0 +1,204 @@
+package enterprise
```

Comment:
I think this package needs to be server if it's in a directory called server.

Reply:
fixed